src/dbos-executor.ts

```diff
@@ -124,9 +124,10 @@ export interface DBOSConfig {
   };
 }
 
-export type DBOSConfigInternal = Omit<DBOSConfig, 'poolConfig' | 'system_database'> & {
+export type DBOSConfigInternal = Omit<DBOSConfig, 'poolConfig' | 'system_database' | 'telemetry'> & {
   poolConfig: PoolConfig;
   system_database: string;
+  telemetry: TelemetryConfig;
```

Comment:
be more strict with this field internally -- it should always be set because we default to "info"